san/sanbase_graphql.py

```diff
@@ -322,29 +335,23 @@ def price_volume_difference(idx, slug, **kwargs):
 
 def eth_top_transactions(idx, slug, **kwargs):
     kwargs = _transform_query_args('eth_top_transactions', **kwargs)
+    return_fields = []
+    for el in kwargs['return_fields']:
+        if isinstance(el,dict):
+            el = str(
+                list(el.keys())[0]
```

Comment:
this weird `[0]` (why ignore all other keys someone without context knowledge my ask) can be avoided by the above suggested dict -> tuple migration